tests/unit/CIGitLogicTest.sh

```diff
@@ -238,7 +238,7 @@ cherry_pick_pr 3
 tag_staging
 
 # Verify output for checklist
-assert_prs_merged_between '1.0.0-0' '1.0.0-2' "[ 3, 1 ]"
+assert_prs_merged_between '1.0.0-0' '1.0.0-2' "[ 1, 3 ]"
```

Comment:
Just FWIW, this didn't just reverse the order. Before, it was in order of most recently merged to least recently merged. Now it's in ascending numerical order, which is much more predictable when looking at actual PR numbers